grunt.js

```diff
@@ -235,7 +235,7 @@ module.exports = function(grunt) {
           transform: [
             require('babelify').configure({
               sourceMapRelative: './src/js',
-              loose: 'all'
+              loose: ['all']
```

Comment:
I don't think this was actually an issue. It seems to work just fine. However, it seems to expect an array, so, I've changed it to be that.